### PR TITLE
Instructions for picking the correct infra. 

### PR DESCRIPTION
**Pull request by jgautam**
https://spectrocloud.atlassian.net/browse/DOC-89

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/584#" title="Author self-approved">jgautam</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/1-new-clusters/05-edge.md [hunk 1]
```diff
@@ -195,6 +195,35 @@ For the OS pack, if a custom CA certificate is required for outgoing traffic, ma
...
+    * Add three disks to the bare-metal machine or the VM instance.
+    * Configure pack settings to use device filter and set up only one OSD per device. As an example, if the disks added were sdd, sde, sdf, the following device filters would need to be set:
+
+**Example:**
+
+          useAllNodes: true
+          useAllDevices: false
+          deviceFilter: ^sd[d-f]
+          config:
+          osdsPerDevice: "1" # this value can be overridden at the node or device levels
```
Comment (brathina-spectro) [inline packages/docs/content/04-clusters/1-new-clusters/05-edge.md:223]: osdsPerDevice should be indented under config

**Review by resmaa — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/584#" title="Author self-approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/584#pullrequestreview-864435989" title="Approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/1-new-clusters/05-edge.md [hunk 2]
```diff
@@ -195,6 +195,35 @@ For the OS pack, if a custom CA certificate is required for outgoing traffic, ma
...
+
+* Virtualized Clusters
+    * In the cluster provisioning flow, we need to add additional disks for all nodes in the worker pool
+    * No changes to the pack settings are required. Ceph cluster config is set to useAllDevices by default
+* Containerized Clusters
+    * Add three disks to the bare-metal machine or the VM instance.
+    * Configure pack settings to use device filter and set up only one OSD per device. As an example, if the disks added were sdd, sde, sdf, the following device filters would need to be set:
+
+**Example:**
+
```
Comment (brathina-spectro) [inline packages/docs/content/04-clusters/1-new-clusters/05-edge.md:218]: This complete block should be under storage. Like 

```
storage: 
  useAllNodes: true
  useAllDevices: false
  deviceFilter: ^sd[d-f]
  config:
    osdsPerDevice: "1"
```

File: packages/docs/content/06-integrations/00-rook-ceph.md [hunk 3]
```diff
@@ -8,31 +8,40 @@ category: ['storage']
...
 
 Rook turns storage software into self-managing, self-scaling, and self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management. Rook uses the facilities provided by the underlying cloud-native container management, scheduling and orchestration platform to perform its duties.
 
+This pack provides configurations for setting up a three-node Ceph cluster (recommended) and a single node Ceph cluster. Please make sure your worker node pool size satisfies the minimum nodes requirement for your Ceph cluster. Additional disks should be attached to your worker pool nodes to deploy a Ceph cluster. Suppose you use existing appliances for your Kubernetes cluster (typical for edge clusters). In that case, you will need to ensure additional disks (1 or 3 - based on your Ceph cluster settings) are attached to the appliance. The device filter needs to be configured in the pack settings for such cases. As an example, if the additional disks were sdd, sde, sdf, the following configuration would be required:
+		
+
+		 **Example:**
```
Comment (brathina-spectro) [inline packages/docs/content/06-integrations/00-rook-ceph.md:27]: Copy the example from above comment

File: packages/docs/content/04-clusters/1-new-clusters/05-edge.md [hunk 4]
```diff
@@ -195,6 +195,36 @@ For the OS pack, if a custom CA certificate is required for outgoing traffic, ma
...
+* Virtualized Clusters
+    * In the cluster provisioning flow, we need to add additional disks for all nodes in the worker pool
+    * No changes to the pack settings are required. Ceph cluster config is set to useAllDevices by default
+* Containerized Clusters
+    * Add three disks to the bare-metal machine or the VM instance.
+    * Configure pack settings to use device filter and set up only one OSD per device. As an example, if the disks added were sdd, sde, sdf, the following device filters would need to be set:
+
+**Example:**
+       
+     storage:
```
Comment (brathina-spectro) [inline packages/docs/content/04-clusters/1-new-clusters/05-edge.md:219]: @resmaa This block needs to be indented as well just like the other one

File: packages/docs/content/06-integrations/00-rook-ceph.md [hunk 5]
```diff
@@ -8,31 +8,42 @@ category: ['storage']
...
+This pack provides configurations for setting up a three-node Ceph cluster (recommended) and a single node Ceph cluster. Please make sure your worker node pool size satisfies the minimum nodes requirement for your Ceph cluster. Additional disks should be attached to your worker pool nodes to deploy a Ceph cluster. Suppose you use existing appliances for your Kubernetes cluster (typical for edge clusters). In that case, you will need to ensure additional disks (1 or 3 - based on your Ceph cluster settings) are attached to the appliance. The device filter needs to be configured in the pack settings for such cases. As an example, if the additional disks were sdd, sde, sdf, the following configuration would be required:
 
+**Example:**		
+```json
+ storage: 
+   useAllNodes: true
+   useAllDevices: false
+   deviceFilter: ^sd[d-f]
+   config:
+      osdsPerDevice: "1" # this value can be overridden at the node or device level
```
Comment (brathina-spectro) [inline packages/docs/content/06-integrations/00-rook-ceph.md:33]: Indentation is wrong, there's an extra space for osdsPerDevice

**Review by truegilb — APPROVED** [review]
Minor edits and approved

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/584#" title="Author self-approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/584#pullrequestreview-864435989" title="Approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/584#pullrequestreview-867425084" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**